openforcefield/tests/test_topology.py

```diff
@@ -618,6 +618,203 @@ def test_to_from_openmm(self):
             assert bond.bond_order == bond_copy.bond_order
             assert bond.bond.is_aromatic == bond_copy.bond.is_aromatic
 
+    @requires_rdkit
+    def test_to_file_vsites(self):
+        """
+        Checks that Topology.to_file() doesn't write vsites
+        """
+        from tempfile import NamedTemporaryFile
+
+        from openforcefield.topology import Molecule, Topology
+
+        mol = Molecule.from_pdb_and_smiles(
+            get_data_file_path("systems/test_systems/1_ethanol.pdb"), "CCO"
+        )
+        carbons = [atom for atom in mol.atoms if atom.atomic_number == 6]
+        positions = mol.conformers[0]
+        mol.add_bond_charge_virtual_site(
+            (carbons[0], carbons[1]),
+            0.1 * unit.angstrom,
+            charge_increments=[0.1, 0.05] * unit.elementary_charge,
+        )
+        topology = Topology()
+        topology.add_molecule(mol)
+        count = 0
+        with NamedTemporaryFile(suffix=".pdb") as iofile:
+            topology.to_file(iofile.name, positions)
+            data = open(iofile.name).readlines()
+            for line in data:
+                if line.startswith("HETATM"):
+                    count = count + 1
+        assert count == 9
+
+    @requires_rdkit
+    def test_to_file_units_check(self):
+        """
+        Checks whether writing pdb with unitless positions, Angstrom positions,
+        nanometer positions, result in the same output
+        """
+        import filecmp
+        from tempfile import NamedTemporaryFile
+
+        from simtk.unit import nanometer
+
+        from openforcefield.tests.test_forcefield import create_ethanol
+        from openforcefield.topology import Molecule, Topology
+
+        topology = Topology()
+        topology.add_molecule(create_ethanol())
+        mol = Molecule.from_pdb_and_smiles(
+            get_data_file_path("systems/test_systems/1_ethanol.pdb"), "CCO"
+        )
```

Comment:
We can probably simplify this by using just one molecule

```suggestion
        mol = Molecule.from_pdb_and_smiles(
            get_data_file_path("systems/test_systems/1_ethanol.pdb"), "CCO"
        )
        topology.add_molecule(mol)
```